package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "resumeker-be",
+  "version": "1.0.0",
+  "description": "🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your portfolio. This is intended to be a guideline. Feel free to add your own flare to it.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Lambda-School-Labs/resumeker-be.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Lambda-School-Labs/resumeker-be/issues"
+  },
+  "homepage": "https://github.com/Lambda-School-Labs/resumeker-be#readme",
+  "dependencies": {
+    "bcrypt": "^4.0.1",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "express-graphql": "^0.9.0",
+    "helmet": "^3.21.3",
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^5.9.4"
```

Comment:
Mongoose/MongoDB is not approved for Lambda Labs, please refer to the engineering standards on what is approved and not approved.

Reply:
Yea... I'll take mogoose out... currently deployed as a postgres db on heroku

Reply:
Mongoose removed, Sequelize ORM added, PG dependency added